src/main/java/com/pnudev/springonlinedelivery/config/Security.java

```diff
@@ -0,0 +1,35 @@
+package com.pnudev.springonlinedelivery.config;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.http.SessionCreationPolicy;
+import org.springframework.security.web.SecurityFilterChain;
+import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter;
+
+@Configuration
+@EnableWebSecurity
+@RequiredArgsConstructor
+public class Security {
+    private final JwtRequestFilter jwtRequestFilter;
+    @Bean
+    public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
+        http
+                .csrf().disable()
+                .formLogin()
+                .and()
+                .authorizeHttpRequests()
+                .requestMatchers("/api/v1/auth/**")
```

Comment:
Did you check if you have the Swagger UI on predefined endpoint? Because I'm not sure that your security config will allow it to run